src/applets/clock/ClockApplet.vala

```diff
@@ -206,6 +138,17 @@ public class ClockApplet : Budgie.Applet {
 			return Gdk.EVENT_STOP;
 		});
 
+		
+
+		// make sure every setting is ready
+		this.update_setting("clock-show-date");
+		this.update_setting("clock-show-seconds");
+		this.update_setting("clock-use-24-hour-time");
```

Comment:
On principle the use of our own 24hr time is nice but IMO we should stick with using the GNOME schema for the clock format (24hr time) for this specifically, as it breaks existing setups otherwise. In our settings UI, we can hook into that schema / GSetting and still expose the option, making it feel like a proper Budgie setting even though it isn't one. I don't really see much value in having a per-applet option for this anyways, most would expect it to apply system wide.